src/assets/wise5/authoringTool/addNode/abstract-import-step/abstract-import-step.component.ts

```diff
@@ -29,7 +31,7 @@ export abstract class AbstractImportStepComponent implements OnInit {
       .copyNodes(nodesToImport, this.importProjectId, this.configService.getProjectId())
       .subscribe((copiedNodes: any[]) => {
         const nodesWithNewNodeIds = this.projectService.getNodesWithNewIds(copiedNodes);
-        this.insertNodesService.insertNodes(nodesWithNewNodeIds, this.targetId);
+        this.insertNodesService.insertNodes(nodesWithNewNodeIds, this.targetId, this.nextId);
```

Comment:
Should we check for ```isFirstNodeInBranchPath(this.nextId)``` here, like we do in ```AddYourOwnNode.submit()```? This way, insertNodeService won't need to depend on CreateNodeBetweenService.

```
if (this.projectService.isFirstNodeInBranchPath(this.nextId)) {
      this.createNodeBetweenService.createNodeBetween(newNode, this.targetId, this.nextId);
    }
```

Reply:
I don't think we can do that since insertNodes() for loops over the nodes.